app/src/test/java/com/kickstarter/viewmodels/ProjectCardholderViewModelTest.java

```diff
@@ -290,7 +304,7 @@ public void testPercentageFundedProgressBarIsGone_projectSuccessful() {
     setUpEnvironment(environment());
 
     this.vm.inputs.configureWith(project);
-    this.percentageFundedProgressBarIsGone.assertValues(true);
+    this.percentageFundedProgressBarIsGone.assertValues(false);
```

Comment:
sweeeeet. we can probably also delete this test and Output™ since we no longer hide the progress bar!